src/App.tsx

```diff
@@ -10,33 +12,26 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState<Tab>(tabs[0]);
+
+  const onTabSelected = (choisedTab: Tab) => {
+    const [currentTab] = tabs.filter(tab => tab.id === choisedTab.id);
+
+    setSelectedTab(currentTab);
+  };
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        Selected tab is
+        {' '}
+        {selectedTab.title}
```

Comment:
Use [interpolation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here to make your code cleaner

Reply:
done